internal/common/agent_util_unix.go

```diff
@@ -66,6 +66,33 @@ func TouchFile(filePathAbsolute string) error {
 	return nil
 }
 
+func printOutputAndError(function string, stdout []byte, err error) {
+	if err == nil {
+		return
+	}
+	stderr := ""
+	ee, ok :=  err.(*exec.ExitError)
+	if ok {
+		stderr = string(ee.Stderr)
```

Comment:
This is kind of cool. `ExitError` is a subtype of `error`, and it has an extra field for the stderr.